pysb/simulator/scipyode.py

```diff
@@ -483,8 +483,18 @@ def __getstate__(self):
         state["_expressions_constant_fn"] = None
         state["_rhs_fn"] = None
         state["_jacobian_fn"] = None
+        # Loggers are not picklable in Python 3.6.
+        state["_logger_extra"] = state["_logger"].extra
```

Comment:
While it would be fine to just store this back in `_logger`, I felt it was nice to be explicit that it's not storing the actual logger.